scss/_patterns_modal.scss

```diff
@@ -49,6 +51,8 @@
     display: flex;
     justify-content: space-between;
     margin-bottom: $spv-inner--small;
+    // add padding to accommodate the width of p-modal__close
+    padding-right: calc(#{$icon-size} + #{$sph-inner--small * 2});
```

Comment:
Both values should be in `rem`, so it should be safe to add them in SCSS and avoid client side calc.

Reply:
Good spot, updated